### PR TITLE
fix for #5061

### PR DESCRIPTION
**Pull request by MaxIsJoe**
I'm marking this as draft to see why Bod did this in the first place
but this fixes teleporters dropping pulled objects when teleporting

### CODE REVIEW

File: UnityProject/Assets/Scripts/Objects/Gateway/TransportUtility.cs [hunk 1]
```diff
@@ -71,22 +71,13 @@ public static void TransportObject(UniversalObjectPhysics objectPhysics, Vector3
...
 				//Transport current
 				TransportObject(currentObj, transportTo, doTileStep, maintRoomChanceModifier);
 
-				if (previous != null && currentObj.gameObject != null)
-				{
-					//There was another object before this one, pulling it. Re-establish pulling. (But only if the current object's gameObject is not null)
-					previous.CmdPullObject(currentObj.gameObject);
-				}
-
 				//TODO: Make pulling acutally continue working across teleporters for clients, not just server
```
Comment (corp-0) [inline UnityProject/Assets/Scripts/Objects/Gateway/TransportUtility.cs:78]: when you pull someone and teleport, you stop pulling on client it seems. That's what I think he meant.

File: UnityProject/Assets/Scripts/Objects/Gateway/TransportUtility.cs [hunk 2]
```diff
@@ -71,22 +71,13 @@ public static void TransportObject(UniversalObjectPhysics objectPhysics, Vector3
...
 				//TODO: Find a way to make the teleporter the teleport not all be on the same tile.
```
Comment (corp-0) [inline UnityProject/Assets/Scripts/Objects/Gateway/TransportUtility.cs:79]: when A and B teleport and A is pulling B, both of them occupy the same tile after teleporting.

Comment (MaxIsJoe) [reply UnityProject/Assets/Scripts/Objects/Gateway/TransportUtility.cs:79]: ...but that's exactly what happens now?

Comment (MaxIsJoe) [reply UnityProject/Assets/Scripts/Objects/Gateway/TransportUtility.cs:79]: nevermind, i double checked; it doesn't.
though i do have to ask, why does it matter? ss13 has similar behavior to what I've made in this PR, and it seems to be fine?

Comment (corp-0) [reply UnityProject/Assets/Scripts/Objects/Gateway/TransportUtility.cs:79]: I'm merely explaining  Bod's comments. I do think it is nice not to break the conga line when teleporting

**Pull request closed without merging**